media-sound/denemo/denemo-2.2.0.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit xdg-utils
+
+DESCRIPTION="A music notation editor"
+HOMEPAGE="http://www.denemo.org/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+ OFL-1.1"
```

Comment:
Still need to fix the license. Looks like I forgot about changing GPL-2+ -> GPL-3+.
Will do this tomorrow.

Reply:
Changed